llvm/test/CodeGen/X86/movmsk-cmp.ll

```diff
@@ -5,6 +5,7 @@
 ; RUN: llc < %s -mtriple=x86_64-unknown-unknown -mattr=+avx2 | FileCheck %s --check-prefixes=AVX,AVX1OR2,AVX2
 ; RUN: llc < %s -mtriple=x86_64-unknown-unknown -mattr=+avx512f | FileCheck %s --check-prefixes=AVX,AVX512,KNL
 ; RUN: llc < %s -mtriple=x86_64-unknown-unknown -mattr=+avx512vl,+avx512dq,+avx512bw | FileCheck %s --check-prefixes=AVX,AVX512,SKX
+; RUN: llc < %s -mtriple=x86_64-unknown-unknown -mattr=+avx10.1-256 | FileCheck %s --check-prefixes=AVX,AVX10-256
```

Comment:
We can merge with a lot of existing checks if we did `--check-prefixes=AVX,AXV512,AVX10-256`

Reply:
The problem here was that if we add `AVX512` to `AVX10-256` then we need to have `KNL` and `SKX` in a separate group otherwise they will duplicate each other. Introduced `EVEX512` for them.